main.cc

```diff
@@ -1193,8 +1195,6 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             // since some features affect storage.
             db::system_keyspace::enable_features_on_startup(feature_service).get();
 
-            db.local().before_schema_keyspace_init();
```

Comment:
Ah, I see you rename it later. Let's put the rename commit first or do the rename and shuffle in the same commit.

Reply:
> This change should be explained in the commit message. 

Sure.

> And maybe the name before_schema_keyspace_init should be changed, because now it must be called even earlier, before system keyspace init, if I understand the intent?

Yes.